### DISCUSSION TITLE
Ports used to Update

### DISCUSSION THREAD

**Discussion by Zagrthos** [AzuraCast/AzuraCast#6603] [Q&A]
Hiya all,

just a quick question regarding the update process of Azura.
I block by default every in- and outgoing connection to the server. Which ports do I have to open so the server can reach it's update host?

As of now following ones are open to go OUT:
53 TC/UDP, 80 TCP, 443 TCP, 8080 TCP

And the following to go IN:
22 TCP, 80 TCP, 443 TCP, 2022 TCP, 8000 TCP

Do I need more?

**Comment by BusterNeece** [comment]
Incoming ports are fine. My assumption is that Docker does all of its transactions over HTTP/HTTPS for pulling new images...I wouldn't expect you to encounter any issues with a setup like that. AzuraCast only makes a single outbound HTTPS request to check for updates and fetch your external IP, but otherwise is entirely self-contained.
